src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -1,18 +1,33 @@
 (ns status-im.contexts.wallet.swap.events
   (:require [re-frame.core :as rf]
             [status-im.constants :as constants]
+            [status-im.contexts.communities.utils :as utils]
             [status-im.contexts.wallet.sheets.network-selection.view :as network-selection]
             [utils.number]))
 
 (rf/reg-event-fx :wallet.swap/start
- (fn [{:keys [_db]}]
+ (fn [{:keys [_]}]
    {:fx [[:dispatch [:open-modal :screen/wallet.swap-select-asset-to-pay]]]}))
```

Comment:
I know this code already existed, but I wonder why it's being used like this.

If we just want to dispatch the modal we don't need use `rf/reg-event-fx` to register a new event, a simple `(rf/dispatch [:open-modal ...])` should be enough.

This caught my attention because the `db` is not being used, creating an event with no DB usage most of the time means it's unnecessary.

nit: `(fn [{:keys[_]}] ,,,)` -> `(fn [_] ,,,)`